tokens/src/mock.rs

```diff
@@ -117,15 +121,23 @@ impl pallet_treasury::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type OnSlash = ();
 	type ProposalBond = ProposalBond;
-	type ProposalBondMinimum = ProposalBondMinimum;
-	type ProposalBondMaximum = ProposalBondMaximum;
-	type SpendPeriod = SpendPeriod;
```

Comment:
remove the unused `ProposalBondMinimum`, `ProposalBondMaximum`, `SpendPeriod`, `MaxApprovals`